lib/main_micro_alp.pm

```diff
@@ -39,6 +39,7 @@ sub load_config_tests {
     loadtest 'transactional/host_config' unless is_dvd;
     loadtest 'rt/rt_is_realtime' if is_rt;
     loadtest 'transactional/enable_selinux' if (get_var('ENABLE_SELINUX') && is_image);
+    loadtest 'transactional/disable_selinux' if (check_var('ENABLE_SELINUX', '0') && is_alp);
```

Comment:
I would not limit this to alp. We can do this also for SLES or SLEM.

Reply:
Done.